src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -134,13 +151,12 @@
 #define VBAT_ADC_PIN            PA0
 #define CURRENT_METER_ADC_PIN   PA1
 #define ADC_INSTANCE            ADC1
-#define CURRENT_METER_SCALE_DEFAULT 2350
+#define CURRENT_METER_SCALE_DEFAULT 1175
```

Comment:
This is a bit of a problem, as this will break the current meter scaling for owners of the existing hardware, and nothing in their documentation will tell them to change it. I think it would be more user friendly if the current meter scale was left as it is, and the documentation for the new hardware would be changed to include instructions on how to set the correct current meter scaling.

Reply:
Ok, now it is roll back to 2350.